### PR TITLE
Eb genotype concordance should read gzipped files

### PR DESCRIPTION
**Pull request by eitanbanks**
(I may have messed up the git merging on this one...)

### CODE REVIEW

File: src/java/picard/vcf/GenotypeConcordance.java [hunk 1]
```diff
@@ -140,8 +140,11 @@ public static void main(final String[] args) {
         }
         if (USE_VCF_INDEX) {
             // Index file is required either because we are using intervals, or because user-set parameter
-            if ((!Tribble.indexFile(TRUTH_VCF).exists() || (!Tribble.indexFile(CALL_VCF).exists()))) {
-                errors.add("Index file(s) not found for one or both of the VCFs.  Note - if intervals are specified, the VCF files must be indexed.");
+            if (!indexExists(TRUTH_VCF)) {
+                errors.add("The index file was not found for the TRUTH VCF.  Note that if intervals are specified, the VCF files must be indexed.");
```
Comment (nh13) [inline src/java/picard/vcf/GenotypeConcordance.java]: Why not check if intervals are specified here too?

Comment (eitanbanks) [inline src/java/picard/vcf/GenotypeConcordance.java]: Because Kylee is taking care of that in her commit.

**Comment by eitanbanks** [comment]
@nh13 checks are still failing.  Is this leftover from my git idiocy?

File: src/java/picard/vcf/GenotypeConcordance.java [hunk 2]
```diff
@@ -152,6 +155,15 @@ public static void main(final String[] args) {
...
     }
 
+    /**
+     * Determines whether an index file exists for the given vcf file using standard extension suffixes
+     *
+     * @param vcf  the vcf file to investigate
+     * @return true if an index file exists, false otherwise
+     */
+    private boolean indexExists(final File vcf) {
+        return !Tribble.indexFile(vcf).exists() && !Tribble.tabixIndexFile(vcf).exists();
```
Comment (nh13) [inline src/java/picard/vcf/GenotypeConcordance.java]: Isn't this logic backwards.  You want to return ```true``` if it exists so it should be ```Tribble.indexFile(vcf).exists() || !Tribble.tabixIndexFile(vcf).exists()```?

**Comment by eitanbanks** [comment]
Okay @nh13 I tried to fix up my branch based on the git magic you taught me today, added back the fix for the logic, rebased, and re-pushed.
Did I do it all correctly?

**Comment by nh13** [comment]
:+1:

**Pull request merged**